spec/suites/layer/TooltipSpec.js

```diff
@@ -184,6 +184,107 @@ describe('Tooltip', function () {
 		expect(group._tooltip._container.innerHTML).to.be("I'm marker 2.");
 	});
 
+	it("it should use a tooltip with a string as content with a FeatureGroup", function () {
+		var marker1 = new L.Marker([55.8, 37.6]);
+		var marker2 = new L.Marker([54.6, 38.2]);
+		var group = new L.FeatureGroup([marker1, marker2]).addTo(map);
+
+		group.bindTooltip("hello");
+
+		// toggle popup on marker1
+		happen.mouseover(marker1._icon, {relatedTarget: map._container});
+		expect(map.hasLayer(group._tooltip)).to.be(true);
+		expect(group._tooltip._container.innerHTML).to.be("hello");
+
+		// toggle popup on marker2
+		happen.mouseover(marker2._icon, {relatedTarget: map._container});
+		expect(map.hasLayer(group._tooltip)).to.be(true);
+		expect(group._tooltip._container.innerHTML).to.be("hello");
+	});
+
+	it("it should use a permanent tooltip with a function as content with a FeatureGroup", function () {
+		var marker1 = new L.Marker([55.8, 37.6], {description: "I'm marker 1."});
+		var marker2 = new L.Marker([54.6, 38.2], {description: "I'm marker 2."});
+		var group = new L.FeatureGroup([marker1, marker2]).addTo(map);
+
+		group.bindTooltip(function (layer) {
+			return layer.options.description;
+		}, {permanent: true});
+
+		expect(map.hasLayer(marker1._tooltip)).to.be(true);
+		expect(marker1._tooltip._container.innerHTML).to.be("I'm marker 1.");
+
+		expect(map.hasLayer(marker2._tooltip)).to.be(true);
+		expect(marker2._tooltip._container.innerHTML).to.be("I'm marker 2.");
```

Comment:
```suggestion
		var group = new L.FeatureGroup([marker1, marker2]);
		group.bindTooltip(function (layer) {
			return layer.options.description;
		}, {permanent: true})

		group.addTo(map);

		expect(map.hasLayer(group._tooltip)).to.be(true);
		expect(group._tooltip._container.innerHTML).to.be("I'm marker 1.");
```
That's how it is untended to be: all FeatureGroup's members share single tooltip.

Not much sense to use with `permanent: true`, though I can imagine a couple of useful use cases.